src/test/java/programminglife/gui/GuiControllerTest.java

```diff
@@ -198,7 +198,10 @@ public void clickOnTest() {
         }
 
         sleep(5, TimeUnit.SECONDS);
+        //backspace to remove the initial values.
+        clickOn("#txtMaxDrawDepth").type(KeyCode.BACK_SPACE);
         clickOn("#txtMaxDrawDepth").type(KeyCode.DIGIT9);
+        clickOn("#txtCenterNode").type(KeyCode.BACK_SPACE);
```

Comment:
This will probably work, but should still not assume anything about the initial value of the field (i.e. also use a loop here)